cdap-app-fabric/pom.xml

```diff
@@ -142,6 +142,18 @@
     </dependency>
     <dependency>
       <groupId>co.cask.cdap</groupId>
+      <artifactId>cdap-api-spark</artifactId>
```

Comment:
or in `cdap-app-fabric-tests`. I would like to keep the main `cdap-app-fabric` clean from Spark.